client/src/components/QandA/AnswerListEntry.jsx

```diff
@@ -1,21 +1,59 @@
 /* eslint-disable react/prop-types */
 import React from 'react';
 import Moment from 'moment';
+import axios from 'axios';
 
-const AnswerListEntry = ({ answer }) => {
-  return (<div className="col-9 text-start">
-    <div key={answer.id}>
-      <div className="row pt-1"><p className="text-wrap">{answer.body}</p></div>
-      <div className="hstack gap-3 fw-light">
-        <div>by {answer.answerer_name}, {Moment(answer.date).format('MMMM Do, YYYY')}</div>
-        <div className="vr"></div>
-        <div>Helpful?</div><div className="text-decoration-underline">Yes</div><div>({answer.helpfulness})</div>
-        <div className="vr"></div>
-        <div className="text-decoration-underline">Report</div>
+class AnswerListEntry extends React.Component {
+  constructor(props) {
+    super(props);
+    const { answer } = this.props;
+    this.state = {
+      answerHelpfulClicked: false,
+      answerHelpfulCount: answer.helpfulness,
+      answerReportClicked: false,
+      reportType: 'button'
+    };
+    this.markAnswer = this.markAnswer.bind(this);
+  }
+
+  markAnswer(e, answer_id) {
+    const handlerType = e.target.id;
+    const { answerHelpfulClicked, answerReportClicked } = this.state;
+    handlerType === 'helpful' && answerHelpfulClicked || handlerType === 'report' && answerReportClicked ?
+      console.log('You can only increment once.') :
```

Comment:
If you meant to leave this message to the user, I suggest changing the console.log to an alert.